Initial Install/Config

baronvonvaderham opened the pull request:
- Initial installation of django and bulletin apps. Configuration of same.
- Removed src directory from tracking, will be created when pip is run on requirements.txt, but associated settings for submodules are included in the settings.py script.
- Needs a working base.html template within django-bulletin
  - Currently have one working locally, will be refined and then commit to that project's repository when completed.
- Needs documentation
  - Have been documenting steps as I experiment.
  - Once site is fully complete, I will duplicate the app in another environment to refine an installation procedure and verify each step.

baronvonvaderham committed:
Initial Installation

Installation of django and bulletin packages and configuration of
same. Loading server using manage.py is successful, but pages will
not render due to missing template base.html.

Will be creating new base template and adding to django-bulletin.

baronvonvaderham committed:
Remove Submodules

Removed src directory from tracking. This will be created locally
after running pip on requirements.txt

jamstooks commented:
Nice work, dude. Couple things...

Change before pull:
- use underscores_for_package_names, not CamelCase

Less important:
- django-overextends may not be necessary
- +ipython and +raven probably aren't required

Also, Bob's working on getting the search functionality into django-bulletin and out of aashe-bulletin, so that'll change things, possibly.

baronvonvaderham committed:
Minor Tweaks

Altered package name to use underscores rather than camel case. Server
initially rejected such formatting.

Removed unnecessary packages from requirements.txt.

rerb commented:
> Bob's working on getting the search functionality into django-bulletin and
> out of aashe-bulletin, so that'll change things,

Might more accurately read "Pushing the search functionality from
aashe-bulletin into django-bulletin is on the list of things Bob has to do
when he has time to devote to the bulletin."  I'm not actually working on
it right now.

On Thu, Jun 25, 2015 at 1:40 PM, Benjamin W Stookey <
notifications@github.com> wrote:

> Nice work, dude. Couple things...
> 
> Change before pull:
> - use underscores_for_package_names, not CamelCase
> 
> Less important:
> - django-overextends may not be necessary
> - +ipython and +raven probably aren't required
> 
> Also, Bob's working on getting the search functionality into
> django-bulletin and out of aashe-bulletin, so that'll change things,
> possibly.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/AASHE/django-bulletin-demo/pull/1#issuecomment-115390721
> .

## 

Robert Erb
Web Application Developer
Association for the Advancement of Sustainability in Higher Education
www.aashe.org | bob.erb@aashe.org | (888) 347-9997 ext 35

## 

Early bird registration http://conference.aashe.org is now open for the _AASHE 
2015 Conference & Expo_, themed Transforming Sustainability Education, in 
Minneapolis, Minn., Oct. 25-28.

Connect with us:   Website http://www.aashe.org/ | Facebook 
https://www.facebook.com/aasheorg | Twitter 
https://twitter.com/AASHENews | LinkedIn 
http://www.linkedin.com/company/aashe

## http://www.aashe.org

Email Confidentiality Notice 
http://www.aashe.org/email_confidentiality_notice

rerb commented:
PR looks clean to me.

baronvonvaderham committed:
Settings Update

Adding static files settings.

baronvonvaderham committed:
manage.py

Re-added manage.py to the branch, after it was mysteriously excluded.

Re-tracked deletions of the DjangoBulletinDemo directory to clean that up.

baronvonvaderham committed:
Big Fixes

Added 'bulletin.tools.plugins' to INSTALLED_APPS to solve problem with
installation not creating DB tables associated with things in that
directory during migration.

Added directory '/templates' and added template files for django-auth.

Altered url patterns to use django-auth as authentication back-end.

Added django-registration to requirements.txt for use in authentication.

Added more settings to settings.py to reflect these changes.
